packages/legend-lego/style/_code-editor.scss

```diff
@@ -13,15 +13,45 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+@use 'mixins' as *;
 
 .code-editor {
-  &__container {
+  height: 100%;
+  width: 100%;
+
+  &__content {
     height: 100%;
```

Comment:
this is insufficient, `hideActionBar` flag should also influence the height, by default when the action bar is shown, the height should be `calc(100% - 2.8rem)`